pkg/model/interface.go

```diff
@@ -21,6 +21,7 @@ type PresetParam struct {
 	GPUCountRequirement       string            // Number of GPUs required for the Preset.
 	TotalGPUMemoryRequirement string            // Total GPU memory required for the Preset.
 	PerGPUMemoryRequirement   string            // GPU memory required per GPU.
+	TuningMethodGPUMemory     map[string]int    // Min GPU memory per tuning method (batch size 1).
```

Comment:
TuningMethodGPUMemory->TuningPerGPUMemoryRequirement

Reply:
Please also update the comments of other requirements in ths struct indicating they are used for inference.

Reply:
Resolved